tools/wptserve/wptserve/request.py

```diff
@@ -308,7 +308,7 @@ def POST(self):
             self.raw_input.seek(0)
             fs = cgi.FieldStorage(fp=self.raw_input,
                                   environ={"REQUEST_METHOD": self.method},
-                                  headers=self.headers,
+                                  headers=self.raw_headers,
```

Comment:
Why the behaviour change here?

Reply:
Because `cgi.FieldStorage` needs to parse the headers and expect the values to be string not bytes. Since `self.readers` is read-only, I figure it would be safe to pass in the original `raw_headers` here.

Reply:
`self.raw_headers` is the headers as `bytes`, no? What type does cgi.FieldStorage expect? `str` on both Py2/3, no?